Gruntfile.js

```diff
@@ -18,11 +18,21 @@ module.exports = function(grunt) {
          target: config.js.all,
       },
 
+      markdownlint: {
+         all: {
+            src: [ '.README.md' ],
```

Comment:
Is the period supposed to be in front of `README.md`?

Reply:
🤦